probe_scraper/fog_checks.py

```diff
@@ -277,7 +277,10 @@ def file_bugs_and_get_emails_for_expiring_metrics(
         addresses = list(metrics_addresses)
 
         filed_bugs: Dict[str, List[str]] = file_bugs(
-            expiring_metrics, latest_nightly_version, bugzilla_api_key, dry_run
+            expiring_metrics,
+            str(int(latest_nightly_version)),
```

Comment:
I'm confused. This won't really change the value. I figured the changes'd be in `fog_checks.file_bugs` to no longer pass `next_version` to `probe_expiry_alert.file_bugs` which, as of #897, assumes it's being called with the latest nightly version, not the one after it.